models/spend_daily.sql

```diff
@@ -0,0 +1,193 @@
+{{ config(materialized='table') }}
+
+with dates as (
+{{ dbt_utils.date_spine(
+        datepart="day",
+        start_date="cast('2019-01-01' as date)",
+        end_date="cast('2025-01-01' as date)"
+    )
+    }}
+),
```

Comment:
since we're adding this package, are we able to replace the CTE i had in `cost_per_query`?

Reply:
The `hours_list` one? Yeah definitely could do